project/Settings.scala

```diff
@@ -0,0 +1,65 @@
+import Dependencies._
+import Merging._
+import Testing._
+import Compiling._
+import sbt.Keys._
+import sbt._
+import sbtassembly.AssemblyPlugin.autoImport._
+
+object Settings {
+
+  val artifactory = "https://artifactory.broadinstitute.org/artifactory/"
+
+  val commonResolvers = List(
+    "artifactory-releases" at artifactory + "libs-release",
+    "artifactory-snapshots" at artifactory + "libs-snapshot"
+  )
+
+  val buildSettings: Seq[_root_.sbt.Def.Setting[_]] = Defaults.defaultSettings ++ Seq(
```

Comment:
Unless there's a conflict, `_root_.sbt.Def.` isn't needed here nor below on line 38.

Also, I'm not sure what `Defaults.defaultSettings` is nor what it does, but its use is deprecated by sbt.

```scala
@deprecated("Default settings split into coreDefaultSettings and IvyModule/JvmModule plugins.", "0.13.2")
lazy val defaultSettings: Seq[Setting[_]] = projectBaseSettings ++ defaultConfigs
```